core/domain/exp_jobs_one_off.py

```diff
@@ -30,10 +30,11 @@
 import feconf
 import utils
 
-(base_models, exp_models,) = models.Registry.import_models([
-    models.NAMES.base_model, models.NAMES.exploration])
+(file_models, base_models, exp_models,) = models.Registry.import_models([
+    models.NAMES.file, models.NAMES.base_model, models.NAMES.exploration])
 
 _COMMIT_TYPE_REVERT = 'revert'
+ALLOWED_IMAGE_EXTENSIONS = ['png', 'jpg', 'gif', 'jpeg']
```

Comment:
Do not copy/paste stuff. There should be a single source of truth for this.

Reply:
Sorry, I didn't get it.

Reply:
Extract the allowed extensions from feconf.ACCEPTED_IMAGE_FORMATS_AND_EXTENSIONS, we need to define them all in one place.

Reply:
Okay

Reply:
Done!